packages/client/src/actions/actions.test.ts

```diff
@@ -65,18 +65,20 @@ describe("actions", () => {
     expectTypeOf<typeof result>().toEqualTypeOf<ActionEditResponse>();
     expect(result).toMatchInlineSnapshot(`
       {
-        "addedLinksCount": 0,
-        "addedObjectCount": 2,
```

Comment:
This count is different. Was the mock creating bad data before?

Reply:
I'm not sure why this is 2, will look into it

Reply:
Looks like the stub was mistakenly returning 1 even when only 1 edit was applied, went ahead and fixed it even though we dont touch it